packages/cypress-angular-dev-server/src/lib/create-angular-webpack-config.ts

```diff
@@ -59,18 +63,26 @@ export async function createAngularWebpackConfig(config: {
       styles,
       polyfills,
       assets,
+      stylePreprocessorOptions,
     }
   );
 
   const webpackConfig = await generateWebpackConfig(
-    getSystemPath(workspaceRoot),
+    getSystemPath(projectRoot),
```

Comment:
Why `projectRoot` here? the first argument expects the `workspaceRoot`.

Reply:
Looking at the existing code it was using the projectRoot even though the variable was named workspaceRoot. On the testing I did using the actual workspaceRoot was working, so Ihave changed it to use workspaceRoot.